README.md

```diff
@@ -38,6 +38,7 @@ Beta:
 * https://www-dev.w3.org/_build_summary.json
 
 Access to https://www-dev.w3.org is restricted by IP.
+**S24 Note: Connect to office VPN**
```

Comment:
As the repo is public I don't think it's a good idea to put S24 specifics in this file.